--- origsrc/liblastfm-0.3.0/admin/qpp	2009-06-04 11:41:27.000000000 -0500
+++ src/liblastfm-0.3.0/admin/qpp	2009-10-02 01:57:02.517246400 -0500
@@ -45,7 +45,7 @@ def find_sources(dir)
       excludes = ['.svn', 'tests', '_build']
       case Platform::IMPL
         when :macosx then excludes << 'win'
-        when :mswin, :cygwin then excludes << 'mac'
+        when :mswin then excludes << 'mac'
         else excludes << 'win' << 'mac'
       end
       Find.prune if excludes.include? File.basename path
@@ -53,7 +53,7 @@ def find_sources(dir)
     elsif File.file? path
       case Platform::IMPL
         when :macosx then next if /_mac\.cpp$/.match path
-        when :mswin, :cygwin then next if /_win\.cpp$/.match path
+        when :mswin then next if /_win\.cpp$/.match path
       end
       yield( path, File.extname( path ) ) unless File.basename(path) == 'EXAMPLE.cpp'
     end
