--- origsrc/liblastfm/admin/findsrc.rb	2011-08-08 14:52:09.000000000 -0500
+++ src/liblastfm/admin/findsrc.rb	2011-08-08 14:54:46.845462500 -0500
@@ -6,7 +6,7 @@ def findsrc dir='.'
   excludes = ['.svn','.git','_include','tests','_build']
   case Platform::IMPL
     when :macosx then excludes<<'win'
-    when :mswin, :cygwin then excludes<<'mac'
+    when :mswin then excludes<<'mac'
     else excludes<<'win'<<'mac'
   end
   
@@ -20,7 +20,7 @@ def findsrc dir='.'
     elsif File.file? path
       case Platform::IMPL
         when :macosx then next if /_mac\.cpp$/.match path
-        when :mswin, :cygwin then next if /_win\.cpp$/.match path
+        when :mswin then next if /_win\.cpp$/.match path
       end
       yield( path, File.extname( path ) ) unless File.basename(path) == 'EXAMPLE.cpp'
     end
@@ -30,4 +30,4 @@ end
 if $0 == __FILE__
   extnames=ARGV.collect {|extname| ".#{extname}"}
   findsrc {|path,extname| puts path if extnames.include? extname}
-end
\ No newline at end of file
+end
