--- origsrc/liblastfm-0.3.0/.qmake.cache	2009-06-04 11:41:27.000000000 -0500
+++ src/liblastfm-0.3.0/.qmake.cache	2009-10-02 01:57:45.183320800 -0500
@@ -26,7 +26,7 @@ CONFIG -= debug_and_release #never works
 *g++* {
     # allow use of 'and', 'or', etc. as symbols
     QMAKE_CXXFLAGS += -fno-operator-names
-    QMAKE_CXXFLAGS_RELEASE += -fvisibility-inlines-hidden -fvisibility=hidden
+    !cygwin-g++:QMAKE_CXXFLAGS_RELEASE += -fvisibility-inlines-hidden -fvisibility=hidden
 }
 
 # look better to Linux folks :P
--- origsrc/liblastfm-0.3.0/src/global.h	2009-06-04 11:41:27.000000000 -0500
+++ src/liblastfm-0.3.0/src/global.h	2009-10-02 01:57:02.532846400 -0500
@@ -29,7 +29,7 @@
     #else
         #define LASTFM_DLLEXPORT __declspec(dllimport)
     #endif
-#elif __GNUC__ >= 4
+#elif __GNUC__ >= 4 && !defined(Q_OS_CYGWIN)
     #define LASTFM_DLLEXPORT __attribute__ ((visibility("default")))
 #else
     #define LASTFM_DLLEXPORT
